nixos/roles/statshost/default.nix

```diff
@@ -627,11 +623,11 @@ in
     })
 
     # outgoing collectd proxy for this location
-    (mkIf (cfgProxyGlobal.enable && statshostService != null) {
+    (mkIf (cfgProxyLocation.enable && statshostService != null) {
       flyingcircus.services.collectdproxy.location = {
         enable = true;
-        statshost = cfgStatsGlobal.hostName;
-        listenAddr = config.networking.hostName;
+        statshost = cfgStats.hostName;
+        listenAddr = (head config.networking.interfaces.ethsrv.ipv6.addresses).address;
```

Comment:
why not `::`?

Reply:
Ok, wildcard seems a not so good idea because the firewall is open. Next best thing: use helper functions from fclib to get the first IPv6 address directly from ENC data. See nixos/roles/mailserver.nix for example.

Reply:
I added a specific ip6tables rule for ethsrv instead of opening the port to everywhere